src/main/java/magicbees/main/Config.java

```diff
@@ -206,6 +209,8 @@ public void setupItems() {
         magicCapsule = new ItemCapsule(CapsuleType.MAGIC, capsuleStackSizeMax);
         voidCapsule = new ItemCapsule(CapsuleType.VOID, capsuleStackSizeMax);
 
+        bloodMagicActive = Loader.isModLoaded("AWWayofTime");
+
```

Comment:
Remove this assignment, it is controlled by the config

Reply:
Can't it's used elsewhere in the mod now